src/pynwb/file.py

```diff
@@ -180,6 +181,7 @@ class NWBFile(MultiContainerInterface):
             {'name': 'session_id', 'type': str, 'doc': 'lab-specific ID for the session', 'default': None},
             {'name': 'institution', 'type': str,
              'doc': 'institution(s) where experiment is performed', 'default': None},
+            {'name': 'keywords', 'type': Iterable, 'doc': 'Terms to search over', 'default': None},
```

Comment:
Instead of ``'type': Iterable`` I believe this should probably be ``'type' : 'array_data'``

Reply:
done